src/main/resources/db/migration/V0.1.1__usage_interval_entities.sql

```diff
@@ -0,0 +1,50 @@
+DO $$
+BEGIN
+  IF NOT EXISTS (SELECT FROM pg_type t INNER JOIN pg_namespace ns ON t.typnamespace = ns.oid WHERE t.typname = 'quality_of_reading' AND ns.nspname = 'usage') THEN
+    CREATE TYPE usage.quality_of_reading AS ENUM (
+      'VALID',
+      'MANUALLY_EDITED',
+      'ESTIMATED_USING_REFERENCE_DAY',
+      'ESTIMATED_USING_LINEAR_INTERPOLATION',
+      'QUESTIONABLE',
+      'DERIVED',
+      'PROJECTED_FORECAST',
+      'MIXED',
+      'RAW',
+      'NORMALIZED_FOR_WEATHER',
+      'OTHER',
+      'VALIDATED',
+      'VERIFIED',
+      'REVENUE_QUALITY'
+    );
+  END IF;
+END $$;
```

Comment:
I like to make my database migration scripts error-free if re-ran on an already migrated database just in case (can sometimes be helpful when working with Flyway). Sometimes, it's easy to make scripts re-runnable with clauses like `IF NOT EXISTS`, but at other times, this wrapping needs to be used.

Reply:
I like it!!